packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -569,80 +646,191 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase {
      * 1 - 100% of the line will be visible
      * @param value
      */
-    public setVisibility(value: number) {
-        this._options.visibility = value;
+    set visibility(value: number) {
+        this._visibility = value;
+    }
+
+    /**
+     * True if dash mode is enabled, false otherwise
+     */
+    get useDash() {
+        return this._useDash;
     }
 
     /**
-     * Turns on/off dashmode
+     * Turns on/off dash mode
      * @param value
      */
-    public setUseDash(value: boolean) {
-        this._options.useDash = value;
+    set useDash(value: boolean) {
+        this._useDash = value;
     }
 
     /**
-     * Sets the dash array.
-     * @param value dash array
+     * Gets the number of dashes in the line
+     */
+    get dashCount() {
+        return this._dashCount;
+    }
+    /**
+     * Sets the number of dashes in the line
+     * @param value dash
      */
-    public setDashCount(value: number) {
-        this._options.dashCount = value;
+    set dashCount(value: number) {
+        this._dashCount = value;
         this._dashArray = 1 / value;
     }
 
+    /**
+     * Sets the dash ratio 0 to 1 (0.5 = half empty, half drawn)
+     */
+    get dashRatio() {
+        return this._dashRatio;

```

Comment:
same here make dashRatio public

Reply:
This one holds a calculated value from the value of `dashCount`